Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -584,18 +521,3 @@ public class FeaturevisorInstance {
     //    }
     //  }
 }
-
-public func createInstance(options: InstanceOptions) -> FeaturevisorInstance? {
```

Comment:
What's your take guys on factory pattern here vs leaving initialisation of the actual object to the final consumer? For me
- Factory could help in the long term (in case we want to do more or other things)
- We could handle some of the init errors and return/throw something more "public"?
- Object init is more transparent to the consumer

Reply:
I missed this diff before.

Personally I am favour of keeping `createInstance`, as API parity is something we wish to maintain across different programming languages.

JavaScript SDK uses `createInstance`, even though internally it's just instantiating the class as `new FeaturevisorInstance()`.

documentation suggests function in case we need to handle more things in future, without breaking the API.

Reply:
Agree, will revert this change and follow this pattern. Make sense to me.